frontend/public/components/factory/list.tsx

```diff
@@ -289,7 +289,7 @@ export const Rows: React.SFC<RowsProps> = (props) => {
       key={key}
       parent={parent}
       rowIndex={index}>
-      <div style={style}>
+      <div style={style} className="co-m-row">
```

Comment:
I suspect there are other tables where we'll want this, but I think this is good for now.